clubs_guides/models.py

```diff
@@ -1,3 +1,55 @@
 from django.db import models
+from django.utils.encoding import python_2_unicode_compatible
 
-# Create your models here.
+
+@python_2_unicode_compatible
+class Category(models.Model):
+    """
+    Category a Mozilla Club's resource guide belongs to
+    """
+    name = models.CharField(max_length=200)
+
+    class Meta:
+        verbose_name_plural = "categories"
+
+    def __str__(self):
+        return str(self.name)
+
+
+@python_2_unicode_compatible
+class ClubsGuide(models.Model):
+    """
+    Resource guides for Mozilla Clubs
+    """
+    title = models.CharField(
+        help_text="The descriptive title for the club guide",
+        max_length=150,
+    )
+    url = models.URLField(
+        help_text="Website link to the guide's content",
+        max_length=500,
+    )
+    language = models.CharField(
+        help_text="The language this guide is in",
+        max_length=150,
+        null=True,
```

Comment:
I think for now it's probably better to leave it as null so that the front-end knows what to show as the main link rather than deciding whether is "en-US" or "American English" or something else.